.travis.yml

```diff
@@ -14,15 +14,15 @@ python:
 - "3.8"
 - "3.7"
 - "3.6"
-- "nightly"  # currently, it's 3.10
```

Comment:
This is 3.10a5 which doesn't have functionality that new `pip` expects from 3.10 so it fails miserably.